.ci/automation-schema.yaml

```diff
@@ -17,6 +17,7 @@ _hook:
   type: str()
   source: str()
   inventory: str(required=False)
+  extra_vars: map(required=False)
```

Comment:
How does this relate to the rest of the PR's contents?

Reply:
Removed, it was actually a conflict with merged content.